bin/inbox-start.py

```diff
@@ -58,11 +58,6 @@
     default=False,
     help="Disables the autoreloader and potentially other non-production features.",
 )
-@click.option(
-    "--enable-tracer/--disable-tracer",
-    default=True,
-    help="Disables the stuck greenlet tracer",
-)
```

Comment:
The tracers are `GreenletTracer` and `KillerGreenletTracer` which are gevent-specific and it does not make sense to have this option anymore.